cfme/automate/simulation.py

```diff
@@ -53,7 +56,11 @@ def simulate(**data):
     Args:
         **data: See :py:data:`sim_form` for keyword reference
     """
-    sel.force_navigate("automate_simulation")
+    try:
```

Comment:
There could be a function decorator if we still have any functions that are not necessary bound to one specific object.